lib/swaps/Swaps.ts

```diff
@@ -629,7 +633,16 @@ class Swaps extends EventEmitter {
     deal.phase = newPhase;
 
     if (deal.phase === SwapDealPhase.AmountReceived) {
-      this.emit('swap.paid', deal);
+      const swapResult = {
+        orderId: deal.orderId,
+        localId: deal.localOrderId,
+        pairId: deal.pairId,
+        amountReceived: deal.makerAmount,
+        amountSent: deal.takerAmount,
+        r_hash: deal.r_hash,
+        peerPubKey: deal.peerPubKey,
+      };
+      this.emit('swap.paid', swapResult, deal.myRole);
```

Comment:
Maybe add `myRole` to `SwapResult` as well, in addition to having it in the grpc response, so the client won't need to guess what's `myRole` from the context (`placeOrder` response vs `subscribeSwaps`)